src/xo-app/menu/index.js

```diff
@@ -136,12 +144,13 @@ export default class Menu extends Component {
 
 const MenuLinkItem = props => {
   const { item } = props
-  const { to, icon, label, subMenu } = item
+  const { to, icon, label, subMenu, pill } = item
 
   return <li className='nav-item xo-menu-item'>
-    <Link activeClassName='active' className='nav-link' to={to}>
-      <Icon icon={`menu-${icon}`} size='lg' fixedWidth />
-      <span className={styles.hiddenCollapsed}>{' '}{_(label)}</span>
+    <Link activeClassName='active' className={classNames('nav-link', styles.centerCollapsed)} to={to}>
+      <Icon className={pill && styles.hiddenCollapsed} icon={`menu-${icon}`} size='lg' fixedWidth />
```

Comment:
Either do `pill ? styles.hiddenCollapsed : null` or `classNames(pill && styles.hiddenCollapsed)` (IMO, the second one is clearer).